cmd/release/cmd/tag.go

```diff
@@ -296,13 +341,13 @@ func init() {
 }
 
 func releaseTypePreRelease(releaseType string) (bool, error) {
-	if releaseType == "rc" || releaseType == "alpha" {
+	if releaseType == "rc" || releaseType == "alpha" || releaseType == "test" {
 		return true, nil
 	}
 
 	if releaseType == "ga" {
 		return false, nil
 	}
 
-	return false, errors.New("release type must be either 'ga', 'alpha' or 'rc', instead got: " + releaseType)
+	return false, errors.New("release type must be either 'ga', 'test', 'alpha' or 'rc', instead got: " + releaseType)
```

Comment:
Depending on how the string is updated in the first comment, that will indicate if this needs to be changed as well.

Reply:
no need for `test` releases, removed it.